.github/workflows/elastic-agent-pull-requests.yml

```diff
@@ -0,0 +1,44 @@
+name: Add to Elastic Agent Data Plane or Control Plane Board
```

Comment:
Nit: If you shorten the title, this looks more descriptive.

![Screenshot 2022-02-04 at 09 35 40](https://user-images.githubusercontent.com/244900/152497446-21be6a5e-f678-4403-a1d2-2e338f3aca0b.png)

Reply:
Yup it keeps only the last run in it.